app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Admin::OrdersController < ApplicationController
+  def index
```

Comment:
Do we need a before action to check either a user is admin or not? if so create an admin controller and get it inherit from application controller and rest of you admin namespace controllers should inherit from it.